x/mint/abci.go

```diff
@@ -59,7 +54,7 @@ func calcTokens(blockTime sdkmath.Uint, minter *types.Minter, maxMintableSeconds
 
 	nsecPassed := calcTimeDifference(blockTime, minter.PrevBlockTimestamp, maxMintableSeconds)
 	if minter.NormTimePassed.LT(types.MonthsInFormula) {
```

Comment:
[everywhere] the term 'norm' has been used as a shorthand form of 'normalized', which meant a time period shrinked by a coeff.
Context: since the initial formula was from 0.47 to 96, not from 0 to 96 we had to represent 1 month as (96 - 0.47)  /96 = 0.9951041666666667. Similarly, we had to transform each period by applying this coefficient, and the result we called 'normalized' period.

Reply:
with the new formula, we do not need this transformation, since the period of the integral matches the real period, so please simplify the code removing the transformations and rename the variables that contain this term

Reply:
With my latest commits, the code got a lot simpler